.github/workflows/ci.yml

```diff
@@ -1385,4 +1628,4 @@ jobs:
           -DCMAKE_FIND_ROOT_PATH_MODE_PACKAGE=ON            \
           -DCppInterOp_DIR="${{ env.CPPINTEROP_BUILD_DIR }}/lib/cmake/CppInterOp"  \
           ..
```

Comment:
I see `micromamba activate CppInterOp-wasm` on line `1616`

Probably that's not required. We would like to stay in the CppInterOp-wasm-build for this !

Reply:
The CppInterOp-wasm build is activate by default when you start an action because of here https://github.com/mcbarton/CppInterOp/blob/027cbb1fc65a39f4b2bb2d2b38fa21c8019b2b57/.github/workflows/ci.yml#L1528 .

Reply:
I shall make the change in my PR as this is more relevant to the shared build.